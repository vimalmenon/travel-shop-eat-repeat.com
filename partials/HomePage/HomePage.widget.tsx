export const HomePage: React.FC = () => {
  return <h1>This is Home Page</h1>;
};
