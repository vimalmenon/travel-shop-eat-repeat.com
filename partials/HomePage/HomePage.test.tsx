import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import { HomePage } from './HomePage.widget';

describe('partials > <HomePage> Component', () => {
  describe('Snapshot Testing', () => {
    it('<HomePage>', () => {
      const wrapper = shallow(<HomePage />);
      expect(toJson(wrapper)).toMatchSnapshot();
    });
  });
});
