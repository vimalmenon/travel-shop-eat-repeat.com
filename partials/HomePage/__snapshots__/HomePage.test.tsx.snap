// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`partials > <HomePage> Component Snapshot Testing <HomePage> 1`] = `
<PageLayout>
  <h1
    className="text-3xl font-bold"
  >
    Work in progress
  </h1>
</PageLayout>
`;
